ui/tests/acceptance/config-ui/messages/messages-test.js

```diff
@@ -46,204 +54,217 @@ module('Acceptance | config-ui', function (hooks) {
     };
     await authPage.login();
   });
+
   hooks.afterEach(async function () {
     await logout.visit();
   });
 
-  test('it should show an empty state when no messages are created', async function (assert) {
-    assert.expect(4);
-    await visit('vault/config-ui/messages');
-    assert.dom('[data-test-component="empty-state"]').exists();
-    assert.dom(PAGE.emptyStateTitle).hasText('No messages yet');
-    await click(PAGE.tab('On login page'));
-    assert.dom('[data-test-component="empty-state"]').exists();
-    assert.dom(PAGE.emptyStateTitle).hasText('No messages yet');
+  test('it should hide the sidebar settings section on community', async function (assert) {
```

Comment:
This test is going to fail in the enterprise test suite, since it will be running on enterprise during that test run. Should probably go ahead and override the enterprise-giving endpoint here to ensure it's consistent between CE and Ent

Reply:
That makes sense! I will update :)